src/components/contact/FormContact.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Group, SimpleGrid, Text, TextInput, Textarea } from "@mantine/core";
+
+const FormContact = () => {
+    return (
+        <form
+        className="contact__form"
+        onSubmit={(event) => event.preventDefault()}>
```

Comment:
Add func on this event : 
```suggestion
        onSubmit={handleSubmitForm}>
```